bee-api/bee-rest-api/src/types/responses.rs

```diff
@@ -135,9 +135,13 @@ pub struct OutputResponse {
     pub output_index: u16,
     #[serde(rename = "isSpent")]
     pub is_spent: bool,
-    pub output: OutputDto,
+    #[serde(rename = "milestoneIndex")]
```

Comment:
Is there a reason we do it for each manually instead of using `#[serde(rename_all = "camelCase")]`?

Reply:
good point

Reply:
i mean we have some fields like "kind" that need rename

Reply:
Yeah in these cases we have to do it manually anyways, also fine to keep it like this for all, since there aren't that many cases

Reply:
So let's keep it as it is for now